StreamChatClient_v3/ChatClient.swift

```diff
@@ -28,7 +28,7 @@ import Foundation
 ///
 public protocol ExtraDataTypes {
     associatedtype User: UserExtraData = NameAndAvatarUserData
-    associatedtype Message: Codable & Hashable = NoExtraMessageData
+    associatedtype Message: MessageExtraData = NoExtraData
     associatedtype Channel: ChannelExtraData = NameAndImageExtraData
```

Comment:
I'm thinking that we can avoid `NameAndAvatarUserData` too and use just `NameAndImageExtraData`.
Most of our SDK's use just image for `User` and `Channel` types.
`avatarURL` looks more related to the `User`, but I think `imageURL` will work too.
Some of customer who is working on backend side on the same time as on mobile clients asked about `image` property and then they realised that we use `avatarURL` for `image` JSON field.
What do you think?

Reply:
OK, makes sense to me 👍

Reply:
fixed in 748d1d7fd5cdc25af2c4c447354e2c9c5cf6636b